### DISCUSSION TITLE
Image Thumbnails For Models and VAE

### DISCUSSION THREAD

**Discussion by HiProAI** [Stability-AI/StableSwarmUI#181] [General]
How do I add an image thumbnail inside of Stable Swarm UI for the Models?
I see the option but it is greyed out!
Perhaps a future version of the software? 
I'd like to use my own images for the models.

**Comment by mcmonkey4eva** [comment]
Step 0: make sure your model is (A) on the machine where swarm is running (not remote) and (B) in `safetensors` format (no `ckpt` - there's a converter under `Utilities` if you have ancient ckpt files)
Step 1: Generate an image you like with the model (or if you're no fun, drag whatever image from your PC into the output area, it'll just snap in)
Step 2: click the hamburger menu next to the model, and select `Edit Metadata`
Step 3: Make sure `Use Image` is checked, feel free to edit any other metadata while you're at it, hit Save.


alternately, if you prefer for some reason, you can also save a `.jpg` or `.png` in the models folder with the same name as the model, and that will be loaded as an icon too. (This is how older WebUIs like auto did things back in the day before safetensors metadata was a good option as it is now)

**Reply by HiProAI** [reply]
I'll give it a try and see how it works out. Thanks
